calibrations/mbd/MBDTRKZ/prorun_mbdtrkz.sh

```
@@ -0,0 +1,128 @@
+else
+  if [[ -n "${_CONDOR_SCRATCH_DIR}" ]]; then
+    time getinputfiles.pl --filelist tracks.list
+  fi
+fi
+
+[[ -s tracks.list ]] || { echo "ERROR: tracks.list is empty" >&2; exit 1; }
+[[ -s fit.list ]] || { echo "ERROR: fit.list is empty" >&2; exit 1; }
+
+tracks_dst_file=$(head -1 tracks.list)
+mbd_dst_file=$(head -1 fit.list)
+echo root.exe -b -q Fun4All_MBD_TrackVertex.C\(${nevents},\"${tracks_dst_file}\",\"${mbd_dst_file}\"\)
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)